docs/DeveloperGuide.md

```diff
@@ -154,6 +154,55 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Payment tracking feature (Amos)
+
+The payment tracking feature is facilitated by `Payment`, `PaymentCommand`[Proposed]
```

Comment:
This section is really well written and detailed, looks very comprehensive and covers all of the commands